### PR TITLE
Implement pop quiz system & fix Netlify issues

### LINKED ISSUE #168: Implement a pop quiz section after the content of each page
A small, javascript pop quiz could be added at the end of each page to reinforce the content covered.
This would probably be around ten (or less) multi choice or true/false questions, which would reveal the answer immediately when they are answered. The answer would not be saved.

A system should be made where any developer can define questions and answers in a simple format (e.g. in markdown), which is then parsed.

An example of this can be seen here:
Screenshot 2020-01-01 at 17 48 06

### PR DESCRIPTION
**Pull request by JoshHeng**
Added a quiz system to the Interactive book. Fixes #168 
See the README for more details or the [binary](https://deploy-preview-180--tender-ardinghelli-ebb79f.netlify.com/docs/binary.html) page for an example

~~This PR also fixes the Netlify issue with links not being relative~~

~~Note that the formatting is still strange for the quiz on the binary page. This is due to a problem with the interactive part above it, which I've already fixed in #174~~

# Screenshot

### CODE REVIEW

**Comment by JoshHeng** [comment]
The new JS doesn't seem to be loading on Netlify. I'll provide screenshots

**Comment by JoshHeng** [comment]
See screenshot

**Comment by JoshHeng** [comment]
I've also fixed the Netlify issue in this PR by changing all links to relative ones (instead of using the `learn.circuitverse.org` prefix).

This means that navigation now works and the correct css and js will be loaded

**Comment by JoshHeng** [comment]
Note that the formatting is still strange for the quiz on the binary page. This is due to a problem with the interactive part above it, which I've already fixed in #174

**Comment by JoshHeng** [comment]
(Preview at [https://deploy-preview-180--tender-ardinghelli-ebb79f.netlify.com/docs/binary.html](https://deploy-preview-180--tender-ardinghelli-ebb79f.netlify.com/docs/binary.html))

**Comment by satu0king** [comment]
Hey, It looks good. However there seems to be a slight UI issue on my system. Can you look into it?

image

**Comment by JoshHeng** [comment]
> Hey, It looks good. However there seems to be a slight UI issue on my system. Can you look into it?
> 
> image

Hi, please refer to the first description/comment of the PR as this issue is addressed

File: _layouts/default.html [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 <html lang="{{ site.lang | default: "en-US" }}">
 {% include head.html %}
-<script src="../assets/js/module.js"></script>
```
Comment (aayush-05) [inline _layouts/default.html]: Any particular reason for removing jquery ?

**Comment by JoshHeng** [comment]
Fixed, please review and merge

Note that currently the styling of the quiz won't work because of another issue which I've fixed in another PR. (#325)

To temporarily bypass this issue, open the inspector and replace `<link rel="stylesheet" href="https://learn.circuitverse.org/assets/css/just-the-docs.css">` with `<link rel="stylesheet" href="/assets/css/just-the-docs.css">`

**Comment by mahmodHammad** [comment]
Cool idea 😀
Does this reusable every where?

**Comment by Shivansh2407** [comment]
> Cool idea 
> Does this reusable every where?

Yes, if this idea is reusable then we can add more quizzes in several pages to make this really interactive. Please confirm this @JoshHeng .

**Comment by JoshHeng** [comment]
> 
> 
> Cool idea 😀
> Does this reusable every where?



> 
> 
> > Cool idea
> > Does this reusable every where?
> 
> Yes, if this idea is reusable then we can add more quizzes in several pages to make this really interactive. Please confirm this @JoshHeng .

Yes, see the README:
```
The quiz should be at the end of a page and is initialised with `{:.quiz}`. Questions are then added in a list, with correct answers being numbered subelements and incorrect answers being unordered subelements. Please see below for an example:
{:.quiz}
1. Example question
	1. Correct answer
	* Incorrect answer
2. What is 1 + 1?
	* 1
	1. 2	
   	* 3
Note that answers must be text only and not contain any other formatting.
```

**Comment by JoshHeng** [comment]
@Shivansh2407 @mahmodHammad

**Comment by Shivansh2407** [comment]
Really nice @JoshHeng 👍

**Pull request merged**